library/ssl_tls12_server.c

```diff
@@ -676,7 +676,11 @@ static int ssl_check_key_curve(mbedtls_pk_context *pk,
                                uint16_t *curves_tls_id)
 {
     uint16_t *curr_tls_id = curves_tls_id;
+#if defined(MBEDTLS_PK_USE_PSA_EC_DATA)
```

Comment:
Suggestion: use `mbedtls_pk_get_group_id()`.

Reply:
Nice! I added that function but I totally forgot about it :D